packages/common/src/crypto/murmurhash.ts

```diff
@@ -0,0 +1,11 @@
+import { MurmurHash3 } from 'murmurhash-wasm';
+
+export class MurmurHash {
+  public static hash(key: string, seed: number): string {
+    const hash = MurmurHash3.hash32(key, seed);
+    const value = hash.readUInt32BE();
+    return value.toString(16);
```

Comment:
as a heads up if you look at the code here:
https://github.com/voiceflow/alexa-service/blob/2417811c80bb57dbd7753314683d7827cd270e52/lib/clients/platform/handlers/capture.ts#L20-L26

We have to make sure that the hash generated is only in an alphabetical or underscore format. I'm not sure if that is always the case for value

Reply:
right now in HEX there are 10 numbers and 6 possible letters. We might need to transform it to be alphabet only

Reply:
fixed

Reply:
fixed now